doc/manual.rst

```diff
@@ -143,28 +143,19 @@ obtained by running *auto-sklearn*. It additionally prints the number of both su
 algorithm runs.
 
 The results obtained from the final ensemble can be printed by calling ``show_models()``. *auto-sklearn* ensemble is composed of scikit-learn models that can be inspected as exemplified by
-`model inspection example <examples/example_get_pipeline_components.html>`_
+`model inspection example <examples/40_advanced/example_get_pipeline_components.html>`_
 .
 
 Parallel computation
 ====================
 
-*auto-sklearn* supports parallel execution by data sharing on a shared file
```

Comment:
The requirement of a shared file system is still there, could you please add this as a requirement? (It's just not used for synchronizing BO any more).